### ISSUE TITLE
WCAG issue - My Media controls disappear at high zoom, narrow browser window, and mobile

### ISSUE THREAD

**Issue by Rainribbon** [Kaltura-EDU/issues-tracking#12]
There has been a long-standing issue in _My Media_ in the LMS where certain controls such as the **Actions** menu and **Analytics**, **Edit**, and **Delete** icons disappear when the user zooms in their browser window to 250%, shrinks the screen width, or uses a mobile device. However, some of the older controls could be accessed with alternative methods, albeit less efficient ones. For example, you could select an individual media entry to access its **Actions** menu and take actions on it.

Now there are two additional buttons that disappear with high zoom/small browser window, "My Recycle Bin" and "My Analytics". There is no alternative way available for users to access the functionality behind these two buttons, creating a failure of [WCAG Success Criterion 1.4.4 AA - Resize Text](https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html).

The following controls should be accessible on the My Media page regardless of viewport size or zoom level:
* Checkboxes to select media entries and associated Actions menu
* My Recycle Bin button
* My Analytics button

If there is a concern about screen real estate on mobile devices, I recommended using a method like Kaltura has on the page of an individual media entry to reduce the number of buttons on the page. On the media entry pages, in mobile view, the "Details" and "Share" tabs are collapsed into a drop-down menu.

Steps to reproduce:
1. Have a KAF where the Recycle Bin and My Analytics have been enabled, or enable them in the KAF.
2. Using a standard browser on a computer, access the LMS with the KAF and go to My Media.
3. Have or create at least 3 media entries in My Media, so you have a few items to take actions on.
4. Zoom in on the browser as far up as it will go (for example, Ctrl + on a PC or Command + on a Mac).
** Expected behavior: All controls are still visible on the page. The Analytics, Edit, and Delete icons continue to display and can be clicked on to access the related feature for the corresponding media entry. You can select media entries with checkboxes and use the Actions menu to access the UI to Publish, Caption & Enrich, or Delete. You can access both the Recycle Bin and My Analytics.
** Actual behavior: Icons, checkboxes, Actions menu, Recycle Bin, and My Analytics all disappear.
5. Using a mobile device, navigate to the LMS in a browser (e.g., Chrome or Safari) and access My Media.
6. Have or create at least 3 media entries there, so you have a few items to take actions on.
7. Note:
** Expected behavior: All controls are still visible on the page. The Analytics, Edit, and Delete icons continue to display and can be clicked on to access the related feature for the corresponding media entry. You can select media entries with checkboxes and use the Actions menu to access the UI to Publish, Caption & Enrich, or Delete. You can access both the Recycle Bin and My Analytics.
** Actual behavior: Icons, checkboxes, Actions menu, Recycle Bin, and My Analytics are not present and you can't use any of this functionality.

Kaltura has filed **ADA-2788 | My Media controls disappear at high zoom, narrow browser width, and mobile view (WCAG 1.4.4 Failure)** for this issue with the following recommendation: "Adopt a responsive design approach similar to individual media entry pages, where controls collapse into a menu for smaller viewports (e.g., mobile). Ensure all controls are accessible regardless of viewport size or zoom level."

**Comment by judyucb** [comment]
Thanks for adding this to Issue Tracker, Tiffany. I actually had missed the issues with My Media page, and will now open my own Case about it. Sigh.

I've had similar Case(s) opened about items disappearing on the Media page (not My Media). Will add a new issue here to record that.
